vocabularies/Analytics.xml

```diff
@@ -176,6 +179,114 @@
+        </Parameter>
+        <Parameter Name="Levels" Type="Collection(Analytics.AutoExpandLevel)" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of aggregation levels forming the visible part of a leveled hierarchy" />
+          <Annotation Term="Core.LongDescription">
+            <String>Each element in the collection defines the property names for one level.
+            A property must not referenced by more than one level.
+            The first element in the collection defines the property names of the coarsest level,
+            the following elements define the property names of consecutively finer-grained aggregation levels.
+            The function result comprises the leveled hierarchy with these levels in preorder.
+            All referenced properties must be groupable.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="LeafLevel" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="A possibly empty set of property names that constitute, together with the property names from `Levels`, the leaf level of the leveled hierarchy" />
+          <Annotation Term="Core.LongDescription">
+            <String>Property names that occur in `Levels` must not be repeated here.
+            All referenced properties must be groupable.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="Aggregation" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="Properties to aggregate for all result entries on all levels" />
+          <Annotation Term="Core.LongDescription">
+            <String>All properties in this collection must be custom aggregates.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="SiblingOrder" Type="Collection(Analytics.AutoExpandSiblingOrder)" Nullable="false">
+          <Annotation Term="Core.Description" String="Sort specification to apply to all direct descendants of a given entry in the resulting leveled hierarchy" />
+        </Parameter>
+        <Parameter Name="BeforeAggregationFilter" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Expression valid for `filter` transformation to restrict the input set before any further procressing" />
+          <Annotation Term="Core.OptionalParameter">
+            <Record/>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="AggregatedValuesLeafFilter" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Expression valid for `filter` transformation to restrict the input set at the most detailed grouping level with conditions on aggregated values" />
+          <Annotation Term="Core.OptionalParameter">
+            <Record/>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="Skip" Type="Edm.Int64" Nullable="false">
```

Comment:
What is the difference compared to $skip, $top, $count? Can't they be used on the result of this function?

Reply:
See https://github.com/SAP/odata-vocabularies/pull/293#discussion_r1475903114

Reply:
OK, I see. Thanks!